tests/integration/test_fim/test_registry/test_nodiff/test_no_diff.py

```diff
@@ -0,0 +1,161 @@
+             os.path.join(key, sub_key_2)]
+
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+reg1, reg2 = test_regs
+
+
+# Configurations
+conf_params = {'WINDOWS_REGISTRY_1':reg1, 'WINDOWS_REGISTRY_2':reg2,
+               'VALUE_1': os.path.join(reg1, no_diff_value), 'VALUE_2' : os.path.join(reg2, no_diff_value),
+               'SREGEX_1': value_sregex, 'SREGEX_2': value_sregex}
+configurations_path = os.path.join(test_data_path, 'wazuh_conf.yaml')
+p, m = generate_params(extra_params=conf_params, modes=['scheduled'])
+
+configurations = load_wazuh_configurations(configurations_path, __name__, params=p, metadata=m)
+
+# Fixtures
+
+@pytest.fixture(scope='module', params=configurations)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@pytest.mark.parametrize('key, subkey, arch, value_name, content, truncated, tags_to_apply', [
+    (key, sub_key_1, KEY_WOW64_64KEY, no_diff_value, "test_content", True, {'no_diff_str'}),
+    (key, sub_key_1, KEY_WOW64_64KEY, "some_value", "test_content", False, {'no_diff_str'}),
+    (key, sub_key_1, KEY_WOW64_32KEY, no_diff_value, "test_content", True, {'no_diff_str'}),
+    (key, sub_key_1, KEY_WOW64_32KEY, "some_value", "test_content", False, {'no_diff_str'}),
+    (key, sub_key_2, KEY_WOW64_64KEY, no_diff_value, "test_content", True, {'no_diff_str'}),
+    (key, sub_key_2, KEY_WOW64_64KEY, "some_value", "test_content", False, {'no_diff_str'})
+
+])
+def test_no_diff_str(key, subkey, arch, value_name, content, truncated, tags_to_apply,
+                  get_configuration, configure_environment, restart_syscheckd,
+                  wait_for_fim_start):
+    """
+    Check the only files detected are those matching the restrict regex
+
+    Parameters
+    ----------
+    key : str
+        Root key (HKEY_*)
+    subkey : str
+        path of the registry.
+    arch : str
+        Architecture of the registry.
+    value_name : str
+        Name of the value that will be created
+    truncated : bool
+        True if an event must be generated, False otherwise.
+    tags_to_apply : set
+        Run test if match with a configuration identifier, skip otherwise.
+    """
+    check_apply_test(tags_to_apply, get_configuration['tags'])
+    values = {value_name: content}
+
+    def report_changes_validator(event):
+        """Validate content_changes attribute exists in the event"""
+        for value in values:
+            folder_str = "{} {}".format("[x32]" if arch == KEY_WOW64_32KEY else "[x64]", sha1(os.path.join(key, subkey).encode()).hexdigest())
+            diff_file = os.path.join(WAZUH_PATH, 'queue', 'diff', 'registry', folder_str, sha1(value.encode()).hexdigest())
+            assert os.path.exists(diff_file), f'{diff_file} does not exist'
+            assert event['data'].get('content_changes') is not None, f'content_changes is empty'
```

Comment:
The format option is not used in these error messages, please remove the f.
```suggestion
            assert os.path.exists(diff_file), '{diff_file} does not exist'
            assert event['data'].get('content_changes') is not None, f'content_changes is empty'
```